grails-app/views/aboutpage.gsp

```diff
@@ -1,130 +1,94 @@
-                                <h5><br>   Groovy School is a series of free weekly training session/workshops where-in attendees are taught the Groovy programming language and some of the tools/frameworks within the Groovy ecosystem. The entire series lasts 6 months and is suitable for people who have never written any code before. At the end of the series, attendees are grouped and assigned an open source project to solve a real world problem affecting the community, using the knowledge and skills that they have obtained during the training.   </h5>
-                                </br>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <div class="section section-dark text-center landing-section">
-                    <div class="container">
-                        <h2>Meet our Instructors</h2>
+            <div class="section landing-section">
+                                <div class="container">
+                                <h3>What is Groovy School?</h3>
+                                <h5>Groovy School is a series of free weekly training session/workshops where-in attendees are taught the Groovy programming language and some of the tools/frameworks within the Groovy ecosystem. The entire series lasts 6 months and is suitable for people who have never written any code before. At the end of the series, attendees are grouped and assigned an open source project to solve a real world problem affecting the community, using the knowledge and skills that they have obtained during the training.</h5>
+                                <br />
+
+                                <h5>We <i class="fa fa-heart icon-danger"></i> &nbsp; what we do.</h5>
+                        <center><h3>CONTRIBUTORS</h3></center>
                         <div class="col-md-4">
                             <div class="team-player">
-                                <img src='${resource(dir: "images/", file: "jenn.jpg")}' alt="Thumbnail Image" class="img-circle img-responsive">
-                                <h5>Jenn Strater<br /><i class="fa fa-twitter"><small class="text-muted"><a href="https://twitter.com/codeJENNerator">@codeJENNerator</i></a></small></h5>
-                                <p>Jenn is a Developer Advocate at Gradle. Also the owner of the Groovy Community Slack;Contributor to Spring REST Docs,Codernarc and more.</p>
+                                <img src='${resource(dir: "images/", file: "placeholder.jpg")}' alt="Thumbnail Image" class="img-circle img-responsive">
+                                <h5>Jenn Strater<br /><small class="text-muted">@twitter</small></h5>
+                                <p>You can write here details about one of your team members. You can give more details about what they do. Feel free to add some <a href="#">links</a> for people to be able to follow them outside the site.</p>
```

Comment:
remove long paragraph text under each contributor. just the photo, the name, and the twitter handle is enough.
Also, add two more rows of contributors so that the total number of contributors is 9